recipe/run_conda_forge_build_setup_osx

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# 1 core available on Travis CI OS X workers: https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
+# MAKEFLAGS is passed through conda build: https://github.com/conda/conda-build/pull/917
+export MAKEFLAGS="-j1 ${MAKEFLAGS}"
```

Comment:
This was not set on the VMs currently. However, I decided to add it explicitly so this is more like what we are doing on Linux. However, as we only get 1 core on Travis CI OS X workers ( [xref]( https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments ) ), we limit it here to 1 core. `conda-build` will automatically pass this through because of PR ( https://github.com/conda/conda-build/pull/917 ), which shows up in `conda-build` 1.20.2. As we are pinned to a lower version currently, we don't use this, but we should be able to use this in the future.